### PR TITLE
fixed the swap issue in time_plot

### PR DESCRIPTION
**Pull request by mts299**
# FIX 

Referencing #348 bug, I made the simple swap intended and then taking @egthomas advice looking for radars that go over the poles; however, only found `rkn` to be a good case for the North and the South currently has another bug to fix. 

I think one can see the change but it is slight in my example. 

## Test code:

**On `develop` branch**:

```
time_plot -png -a -mag -latmin 72  20100305.0201.00.rkn.fitacf > 20100305.0201.00.rkn.dev.mag.png
```

20100305 0201 00 rkn mag dev

**On  `FIX/time_plot` branch**:

```
time_plot -png -a -mag -latmin 72  20100305.0201.00.rkn.fitacf > 20100305.0201.00.rkn.swp.mag.png
```

20100305 0201 00 rkn swp mag

I notice a slight difference with a bit more data with the swap fix.

### CODE REVIEW

**Comment by mts299** [comment]
@ecbland and @egthomas does one of you want to test this PR?

File: codebase/superdarn/src.bin/tk/plot/time_plot.1.8/time_plot.c [hunk 1]
```diff
@@ -1413,7 +1413,7 @@ int main(int argc,char *argv[]) {
         if ((geoflg) || (magflg)) {
           // tmp was also used for a char array above this is why its bad to define within a if function
```
Comment (ecbland) [inline codebase/superdarn/src.bin/tk/plot/time_plot.1.8/time_plot.c:1414]:
```suggestion
```

File: codebase/superdarn/src.bin/tk/plot/time_plot.1.8/time_plot.c [hunk 2]
```diff
@@ -1427,9 +1427,9 @@ int main(int argc,char *argv[]) {
 
           if (tlat<blat) {
             // TODO: this isn't actually swapping things
```
Comment (ecbland) [inline codebase/superdarn/src.bin/tk/plot/time_plot.1.8/time_plot.c:1429]:
```suggestion
```

**Review by ecbland — APPROVED** [review]
Apologies to @mts299 for neglecting this PR for so long! I finally found some data to test it properly. You can see the difference nicely in the following plots from SPS:
```
time_plot -ex 24:00 -v -gs -mag -latmin -80 -latmax -75 -png -b 0 20180306.sps.fitacf3 > plot.png
```

On `develop` :
dev

On `FIX/time_plot`:
fix

So this PR fixes the bug. I suggest that we remove some outdated comments in the code, and then I will merge this. Thanks @mts299!

**Pull request merged**